src/stories/Table/Table.stories.tsx

```diff
@@ -0,0 +1,98 @@
+
+import { ComponentPropsWithoutRef, useState } from 'react';
+import { Meta, Story } from '@storybook/react/types-6-0';
+import Button from 'src/Button/Button';
+import TableComponent, { TableProps } from 'src/Table';
+import DocBlock from '.storybook/DocBlock';
+
+export default {
+    title: 'Lakefront/Table',
+    component: TableComponent,
+    parameters: {
+        docs: {
+            page: DocBlock
+        }
+    }
+} as Meta;
+
+const columns = [
+    {
+        Header: 'RELEASE',
+        accessor: 'release',
+        Cell: ({ cell: { value } }) => value
+    },
+    {
+        Header: 'COUNT',
+        accessor: 'unique_issues'
+    },
+    {
+        Header: 'KM',
+        accessor: 'autonomous_km'
+    },
+    {
+        Header: 'KM / ISSUE',
+        accessor: 'km_disengagement',
+        Cell: ({ cell: { value } }) => value?.toFixed(4) || ''
+    },
+    {
+        Header: 'ISSUE / 100 KM',
+        accessor: 'issue_km',
+        Cell: ({ cell: { value } }) => value?.toFixed(4) || ''
+    }
+];
+
+const customData = [{ release: 'r2204_1_0', unique_issues: 24, autonomous_km: 166.992, km_disengagement: 6.9579999999, issue_km: 0.14371985 },
```

Comment:
I can see this isn't real data, and that it's a nice demo of what the table can do, but I think it's probably best to pick some other topic area to use to demonstrate the component in order to avoid any implication that it's representative of how our stack is performing.  Perhaps rename the columns to something more generic?

Reply:
Given generic column name.